save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt

```diff
@@ -53,35 +51,14 @@ val demoView: VFC = VFC {
  * [VFC] for demo view (a temporary workaround for diktat)
  */
 val diktatDemoView: VFC = VFC {
-    val demoView = demoView()
-    demoView {
+    commonDemoView {
         emptyDemoRunRequest = DemoRunRequest.diktatDemoRunRequest
         demoRunEndpoint = "/diktat/run"
         configName = "diktat-analysis.xml"
     }
 }
 
-/**
- * [Props] for DemoView
- */
-external interface DemoViewProps : Props {
-    /**
-     * An initial value of [DemoRunRequest]
-     */
-    var emptyDemoRunRequest: DemoRunRequest
-
-    /**
-     * Endpoint to run this demo
-     */
-    var demoRunEndpoint: String
-
-    /**
-     * Optional config name for this demo
-     */
-    var configName: String?
-}
-
-private fun demoView(): FC<DemoViewProps> = FC { props ->
+private val commonDemoView: FC<DemoViewProps> = FC { props ->
```

Comment:
so, `val` is better? not a `fun`?

Reply:
making `FC` be `val` allows to remove line 
```
val commonDemoView = demoView()
```

Actually there is no other difference

Reply:
> Actually there is no other difference

So it's only refactoring, isn't it?

Reply:
We used to create a new demoView instance, which was recreated during each re-render (and that caused no result on frontend despite the fact that the result was received). Now we use commonDemoView that is created once.

Reply:
so `val` is better :)